dsl_parser/elements/imports.py

```diff
@@ -113,20 +112,18 @@ def parse(self,
                 resources_base_path=resources_base_path)
             slash_index = blueprint_location.rfind('/')
             self.resource_base = blueprint_location[:slash_index]
-        imports_list, parsed_blueprint = _combine_imports(
+        parsed_blueprint = _combine_imports(
             parsed_dsl_holder=main_blueprint_holder,
             dsl_location=blueprint_location,
             resources_base_path=resources_base_path,
             version=version,
             resolver=resolver,
             validate_version=validate_version)
-        self.imported = imports_list
         return parsed_blueprint
 
     def calculate_provided(self, **kwargs):
         return {
-            'resource_base': self.resource_base,
-            constants.IMPORTED: self.imported
+            'resource_base': self.resource_base
```

Comment:
If we're returning a single value, can we just return it, instead of a dict?

Reply:
Sadly no, this is a part of infrastructure